src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -157,39 +157,37 @@
  * Example with default values:
  * </p>
  * <pre>
- * public class MyClass {
- * private int myPrivateField1;
+ * class Example1 {
```

Comment:
Why did we drop `public` access modifier in all examples?

Reply:
There is no reason to have. It works same with and without.

I think we will drop class definition also after we implement start-finish comments in Examples to let us define what should be extracted from real java file.

Reply:
Yes, I went for shortness. I don't see a reason for keeping the access modifier unless the check requires it to be there.

Reply:
I can agree with brevity of examples; however, going forward please mention these sorts of changes in review comments so that we can shorten the feedback loop.